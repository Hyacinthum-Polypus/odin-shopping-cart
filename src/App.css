html, body {
  min-height: 100vh;
  height: auto;
  margin: 0;
  padding: 0;
}

#root {
  min-height: 100vh;
  height: auto;
}

* {
  font-family: sans-serif;
  box-sizing: border-box;
}

.app-body {
  padding: 25px;;
  background-color: #588cbf;
  min-height: 100vh;
  height: auto;
}
