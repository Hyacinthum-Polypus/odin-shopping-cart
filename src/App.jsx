import NavBar from './components/NavBar'

const App = () => {
  
  return (
    <>
      <NavBar />
      <h1>Homepage</h1>
      <p>Lorem ipsum sorrum soe rum gaah goo goo gahh gahh waa wahh wahh goo gahh</p>
    </>
  )
}

export default App
