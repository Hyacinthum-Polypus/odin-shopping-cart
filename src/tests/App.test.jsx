import { describe, it, expect } from "vitest";
import { render, screen } from "@testing-library/react";
import userEvent from "@testing-library/user-event";
import App from "../App.jsx";

describe("App component", () => {
    it("renders homepage", () => {
        const { container } = render(<App />);
        expect( container ).toMatchSnapshot();
    });
    

    
});