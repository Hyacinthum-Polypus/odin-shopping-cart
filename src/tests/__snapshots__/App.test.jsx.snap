// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App component > renders homepage 1`] = `
<div>
  <h1>
    Homepage
  </h1>
</div>
`;
